c-pallets/file-bank/src/lib.rs

```diff
@@ -355,13 +363,13 @@ pub mod pallet {
 		#[pallet::weight(<T as pallet::Config>::WeightInfo::upload_declaration())]
 		pub fn upload_declaration(
 			origin: OriginFor<T>,
-			file_hash: Vec<u8>,
+			file_hash: Vec<u8>, // Hashes have a fixed size, Vec<u8> doesn't seem the best choice
```

Comment:
I define a type for it:
https://github.com/CESSProject/cess/pull/88/files#diff-9c37b1a61a6e9394aac1e9383bf9d3fac9539378d1af8fcd75aed4f8a4f42967R388
defined here:
https://github.com/CESSProject/cess/pull/88/files#diff-1b48106f9cdd51de414e128717cc6b39dc2059cc0f9087f3c456ce63c2e4c6d1R12